### ISSUE TITLE
Intel Arc A750 QSV: Blue square artifacts in AV1 recordings

### ISSUE THREAD

**Issue by tyisafk** [obsproject/obs-studio#10362]
### Operating System Info

Windows 11

### OBS Studio Version

30.1.0-rc1

### OBS Studio Log URL

https://obsproject.com/logs/KOdR8dTzk0N0VKLe

### Expected Behavior

No artifacts

### Current Behavior

When recording using QSV AV1, I get a square artifacts that pop up for a frame every few seconds.  This only happens in OBS.  They do not appear when recording QSV AV1 via XSplit, converting a file to AV1 via Handbrake, or rendering an AV1 file in Resolve.  I've already tried to DDU and reinstall my drivers.  They can be hard to spot without going frame by frame, but in my peripheral vision they're like a blip that causes eye strain.  I've tried CBR, CQP, and ICQ.  H264 and HEVC do not have this issue.

![image](https://github.com/obsproject/obs-studio/assets/13793605/d13a2fcd-bb8b-4710-9055-a904b91b526e)

GPU: Intel Arc A750

### Steps to Reproduce

1. Select QSV AV1
2. Start recording

**Comment by RytoEX** [comment]
Does this also happen with OBS Studio 30.0.2? Please provide a sample video clip.

**Comment by RytoEX** [comment]
I am so far unable to confirm this on my Intel system (Arc A770M with driver 31.0.101.5085). We require more information to look into this further. Please see my questions in my previous reply.

**Comment by karakaaaa** [comment]
yeah If anyone could please help with this im having the same issue on my asrock a77016gb its but its all over the screen I cannot capture it in a screen shot because they do not appear in screen shots or when I pause the video

**Comment by derrod** [comment]
This increasingly sounds like a decoder or hardware issue, please provide samples of AV1 videos where you encounter these problems.

**Comment by WeiZTD** [comment]
Related [support ticket](https://discord.com/channels/554824368740630529/1258445691856879677) on [Intel Insiders Community](https://discord.com/invite/intel) Discord

**Comment by RytoEX** [comment]
I've reviewed the thread on the Intel Insiders Community Discord server, and I do not agree with the conclusion that this is an OBS bug since another user reported:
> Same problem, on EA FC 24, with Windows GameBar Recorder

If it also happens with recordings made with Windows Game Bar Recorder, it's unlikely to be our bug.
